gtsam/nonlinear/nonlinear.i

```diff
@@ -824,4 +824,17 @@ virtual class NonlinearEquality : gtsam::NoiseModelFactor {
   void serialize() const;
 };
 
+template <T = {gtsam::Point2, gtsam::StereoPoint2, gtsam::Point3, gtsam::Rot2,
+               gtsam::SO3, gtsam::SO4, gtsam::SOn, gtsam::Rot3, gtsam::Pose2,
+               gtsam::Pose3, gtsam::Cal3_S2, gtsam::CalibratedCamera,
+               gtsam::PinholeCamera<gtsam::Cal3_S2>,
```

Comment:
I think there could be a comment (or note) on what will be the name for this?

Reply:
It will be `NonlinearEquality2PinholeCameraCal3_S2`. This should maybe go in `gtwrap`'s docs?

Reply:
Yeah I agree